build/web_compose/compose.sh

```diff
@@ -15,36 +15,38 @@
 
 export MIXER_API_KEY=$DC_API_KEY
 
+if [[ $USE_SQLITE == "true" ]] then
+    export USER_DATA_PATH=/userdata/
+    export SQLITE_PATH=/sqlite/datacommons.db
+else
+    export USER_DATA_PATH=$GCS_DATA_PATH
+fi
+
 /go/bin/mixer \
     --use_bigquery=false \
     --use_base_bigtable=false \
     --use_custom_bigtable=false \
     --use_branch_bigtable=false \
-    --sqlite_path=$SQL_DATA_PATH/data/datacommons.db \
+    --sqlite_path=$SQLITE_PATH \
```

Comment:
Assuming mixer is ok if `sqlite_path` is empty.

Reply:
Mixer would only use sqlite_path if use_sqlite == true, so it's fine to be empty.